blocks/lead-art-block/features/leadart/default.jsx

```diff
@@ -215,7 +225,7 @@ class LeadArt extends Component {
             controlsFont={getThemeStyle(arcSite)['primary-font-family']}
             autoplayPhrase={this.phrases.t('global.gallery-autoplay-button')}
             pausePhrase={this.phrases.t('global.gallery-pause-autoplay-button')}
-            pageCountPhrase={(current, total) => this.phrases.t('global.gallery-page-count-text', { current, total })}
+            pageCountPhrase={/* istanbul ignore next */ (current, total) => this.phrases.t('global.gallery-page-count-text', { current, total })}
```

Comment:
ignored as this is testing the usage of another library and the translations count here